### DISCUSSION TITLE
Next.js Notion Starter Kit

### DISCUSSION THREAD

**Discussion by giscus** [giscus/giscus#1160] [General]
# Next.js Notion Starter Kit

Deploy your own Notion-powered website in minutes with Next.js and Vercel.

http://localhost:3000/nextjs-notion-starter-kit-27aaa1b503fb44e9a0d842a9e8f39325

**Comment by sgamerw** [comment]
1

**Reply by laymonage** [reply]
Please stop spamming this repo with your discussions. Use your own repository for testing.

**Reply by sgamerw** [reply]
@laymonage sorry to spam this repo, already changed the config of Next.js Notion Starter Kit.

**Reply by laymonage** [reply]
No worries, thank you for volunteering to make giscus available in more projects! 🙌
